Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIWebNavigationCocoa.mm

```diff
@@ -32,13 +32,60 @@
 
 #if ENABLE(WK_WEB_EXTENSIONS)
 
+#import "WKFrameInfo.h"
+#import "WKWebViewPrivate.h"
 #import "WebExtensionFrameIdentifier.h"
 #import "WebExtensionFrameParameters.h"
 #import "WebExtensionTab.h"
 #import "WebFrame.h"
+#import "_WKFrameTreeNode.h"
+#import <wtf/BlockPtr.h>
 
 namespace WebKit {
 
+static WebExtensionFrameParameters frameParametersForFrame(_WKFrameTreeNode *frame, _WKFrameTreeNode *parentFrame, WebExtensionTab* tab, WebExtensionContext* extensionContext, BOOL includeFrameIdentifier)
```

Comment:
`bool` instead of `BOOL`.

Reply:
I wasn't sure since this was an Obj-C++ file. I'll go with bool.

Reply:
I'm pretty sure `BOOL` is `bool` in ObjC++, but just to be sure.

`bool` has better type conversion semantics, since returning a pointer will properly convert to 0 or 1, when compared to `BOOL` which typically requires `!!` for pointers to not be a ticking time bomb with the bit lottery.